openpype/plugins/publish/extract_review_slate.py

```diff
@@ -314,6 +308,41 @@ def process(self, instance):
                 slate_subprocess_cmd, shell=True, logger=self.log
             )
 
+            # Create slate with silent audio track
+            if input_audio:
+                # silent slate output path
+                slate_silent_path = slate_path.replace(".png", "Silent" + ext)
+                _remove_at_end.append(slate_silent_path)
+
+                slate_silent_args = [
+                    ffmpeg_path,
+                    "-i", slate_v_path,
+                    "-f", "lavfi", "-i",
+                    "anullsrc=r={}:cl={}:d={}".format(
+                        audio_sample_rate,
+                        audio_channel_layout,
+                        one_frame_duration
+                    ),
+                    "-c:v", "copy",
+                    "-c:a", audio_codec,
+                    "-map", "0:v",
+                    "-map", "1:a",
+                    "-shortest",
+                    "-y",
+                    slate_silent_path
+                ]
+                slate_silent_subprocess_cmd = " ".join(slate_silent_args)
+                # run slate generation subprocess
+                self.log.debug(
+                    "Silent Slate Executing: {}".format(
+                        slate_silent_subprocess_cmd))
+                openpype.api.run_subprocess(
+                    slate_silent_subprocess_cmd, shell=True, logger=self.log
+                )
+
+                # replace slate with silent slate for concat
+                slate_v_path = slate_silent_path
```

Comment:
The source `slave_v_path` should be probably removed?

Reply:
Not sure if I follow. `slate_v_path`  is used for encoding the slate video. If audio is present in the main review, `slate_v_path` is used as a source for `slate_silent_path`. In the following code, concat needs to use either `slate_v_path` or `slate_silent_path` depending on if audio is present or not. Therefore I overwritten the `slate_v_path` by `slate_silent_path` in the end of `if input_audio` block.

Reply:
No, you change the filepath to `{dir}/{basename}Silent{ext}` (https://github.com/pypeclub/OpenPype/pull/3162/files#r870019552).

Reply:
Feeling more dumb than normal today. IMHO it is cleaned together with other in-betweens when not needed:
https://github.com/pypeclub/OpenPype/blob/b0c9e8a29d4c16d3276989fee37c8d8762d67809/openpype/plugins/publish/extract_review_slate.py#L414

'cause `slate_v_path` is appended to _remove_at_end
https://github.com/pypeclub/OpenPype/blob/b0c9e8a29d4c16d3276989fee37c8d8762d67809/openpype/plugins/publish/extract_review_slate.py#L294

I just tested, and found no leftovers after publish